### PR TITLE
Added number of bytes for token to config

### PR DESCRIPTION
**Pull request by ClaytonPassmore**
Forgot to add this on initial server commit.

### CODE REVIEW

**Comment by ClaytonPassmore** [comment]
@henryc132 can you take a look?

File: Server/App/config.py [hunk 1]
```diff
@@ -45,3 +45,4 @@ class FacebookConfig(object):
 class SecurityConfig(object):
     ACCESS_TOKEN_HEADER = 'X-CUE-ACCESS-TOKEN'
     USER_ID_HEADER = 'X-CUE-USER-ID'
+    ACCESS_TOKEN_RAW_BYTES = 120
```
Comment (henryc132) [inline Server/App/config.py:48]: Is FB access token only 120 bytes, thought it was larger

Comment (henryc132) [reply Server/App/config.py:48]: Actually I guess you mean the access token we generate to give back to user, but that is a 160 character VARCHAR in schema?

**Review by henryc132 — APPROVED** [review]

File: Server/App/config.py (hunk 1 above)
Comment (ClaytonPassmore) [reply Server/App/config.py:48]: Yeah so it generates 120 bytes but when you encode it in base 64 it ends up being 160

**Pull request merged**